build.gradle

```diff
@@ -5,15 +5,32 @@ plugins {
 group 'org.example'
 version '1.0-SNAPSHOT'
 
+compileJava {
+    sourceCompatibility = 17
+    targetCompatibility = 17
+}
+
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.2'
-    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.2'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.2'
```

Comment:
Please group your implementation and test specific dependencies

Reply:
grouped